hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -20,6 +20,7 @@
 
 import org.apache.hadoop.classification.InterfaceAudience;
 import org.apache.hadoop.classification.InterfaceStability;
+import org.apache.hadoop.security.ssl.DelegatingSSLSocketFactory;
```

Comment:
there's a risk that because this class wil load iff wildfly is on the CP, then you may not even be able to refer to constants here without it being present. I's rather the fix there than in this class though